dexs/injective/index.ts

```diff
@@ -0,0 +1,51 @@
+import { httpGet } from "../../utils/fetchURL";
+import { FetchResultVolume, SimpleAdapter } from "../../adapters/types";
+
+interface IVolumeResponse {
+  total_volume_usd: number;
+}
+
+interface IOpenInterestResponse {
+  total_open_interest: number;
+}
+
+const DERIVATIVE_URL = `https://bigquery-api-636134865280.europe-west1.run.app/injective_derivative_volume`;
```

Comment:
thanks for the PR @manualy 

 - what exactly is included as spot and derivatives volume? feel like this is duplicating what we are tracking for helix, if it is, skip this adapter all together
 - If not, can you split it into two adapters
 - the api returns only cumulative data? if so, shouldnt previous record be fetched and subtracted?